features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures/postgresql/postgre-11x/sessiondata-cleanup/postgresql_11-session-data-cleanup.sql

```diff
@@ -340,21 +389,65 @@ LOOP
         END IF;
 
         -- Deleting user-session mappings from 'idn_auth_user_session_mapping' table
-        IF (enableLog AND logLevel IN ('TRACE')) THEN
-        notice := 'BATCH DELETE START ON TABLE '||purgingSessionUserMappingTable||' WITH :'||batchCount;
-        RAISE NOTICE '%',notice;
-        END IF;
+        EXECUTE 'SELECT count(1) from pg_catalog.pg_tables WHERE schemaname = current_schema() AND tablename =  $1' into rowcount USING purgingSessionUserMappingTable;
+        IF (rowcount = 1)
+        THEN
+            IF (enableLog AND logLevel IN ('TRACE')) THEN
+                notice := 'BATCH DELETE START ON TABLE '||purgingSessionUserMappingTable||' WITH :'||batchCount;
+                RAISE NOTICE '%',notice;
+            END IF;
 
-        EXECUTE 'DELETE from '||quote_ident(purgingSessionUserMappingTable)||' a USING '||purgingBatchTable||' b where a.'||purgeBseColmn||' = b.'||purgeBseColmn||'';
-        GET diagnostics deleteMappingCount := ROW_COUNT;
-        COMMIT;
+            EXECUTE 'DELETE from '||quote_ident(purgingSessionUserMappingTable)||' a USING '||purgingBatchTable||' b where a.'||purgeBseColmn||' = b.'||purgeBseColmn||'';
```

Comment:
Do we need this space?

Reply:
Addressed in https://github.com/wso2/carbon-identity-framework/pull/3178/commits/3d2233c7745ad78dc3f6e7e384b163ed21b9ddaa